build/gradle_test.go

```
@@ -0,0 +1,24 @@
+package build
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDownloadExtractorsFromReleases(t *testing.T) {
+	tempDirPath, err := fileutils.CreateTempDir()
+	assert.NoError(t, err)
+	defer fileutils.RemoveTempDir(tempDirPath)
```

Let's not ignore the error returned from ```fileutils.RemoveTempDir```.